CHANGELOG.md

```diff
@@ -3,7 +3,11 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
-## [unreleased]
+## unreleased
+### Changed
+* Migrated completely off of bower in favor of npm
+
+## [6.0.x](https://github.com/apache/trafficcontrol/tree/6.0.x) - branched
```

Comment:
Put it in the current *Changed* changelog section, I'll take care of the 6.0.0 part.